website/server/middlewares/cron.js

```diff
@@ -6,9 +6,10 @@ import Bluebird from 'bluebird';
 import { model as Group } from '../models/group';
 import { model as User } from '../models/user';
 import { recoverCron, cron } from '../libs/cron';
-import { v4 as uuid } from 'uuid';
 
 const daysSince = common.daysSince;
+// Wait this length of time in ms before attempting another cron
+const CRON_TIMEOUT = new Date(60 * 60 * 1000).getTime();
```

Comment:
Suggestion: rename variable to something more descriptive.

Ideas: CRON_TIMEOUT_WINDOW, CRON_TIMEOUT_MS, CRON_TIMEOUT_WAIT, CRON_TIMEOUT_MAX_WAIT, etc.